tests/helper/reporter/httpMeasurementReporter.go

```diff
@@ -0,0 +1,81 @@
+package reporter
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/onsi/ginkgo/config"
+	"github.com/onsi/ginkgo/types"
+
+	. "github.com/onsi/ginkgo"
+)
+
+type HTTPMeasurementReporter struct {
+	url string
+}
+
+func NewHTTPMeasurementReporter(url string) *HTTPMeasurementReporter {
+	r := HTTPMeasurementReporter{
+		url: url,
+	}
+	return &r
+}
+
+func (r *HTTPMeasurementReporter) SpecSuiteWillBegin(config config.GinkgoConfigType, summary *types.SuiteSummary) {
+}
+
+func (r *HTTPMeasurementReporter) BeforeSuiteDidRun(setupSummary *types.SetupSummary) {}
```

Comment:
Do you keep it intentionally for future enhancement ?

Reply:
It is required as otherwise, the `HTTPMeasurementReporter` wouldn't be implementing the required interface.

Reply:
Ahh..right
Thanks for the details